forge-game/src/main/java/forge/game/ability/AbilityFactory.java

```diff
@@ -352,6 +352,11 @@ private static final TargetRestrictions readTarget(Map<String, String> mapParams
             prompt = "Select target " + tgtWhat;
         }
 
+        if (mapParams.containsKey("SpreeCost")) {
```

Comment:
hmn I don't understand...
If they don't have SpellDescription the card would look wonky anyway:
Or are you thinking about a sub only - then a fallback might be better?

Reply:
Sorry, I still don't understand why Spree needs special treament?

It already cuts off the already targeted parts, so now we'd be showing duplicated text for no reason?

![grafik](https://github.com/Card-Forge/forge/assets/8506892/76ca12bf-c022-4d24-b239-e4b593662498)

Reply:
I'm glad you keep asking questions, don't want to change anything if it doesn't make sense.

I wired it up because unlike you I haven't selected this pref
<img width="680" alt="image" src="https://github.com/Card-Forge/forge/assets/103371817/3602a6a5-3849-4ad2-b19a-bc3bc34c73a8">
Without that pref things can be a bit more confusing
I believe that pref is off by default / on a new install

I'm not sure which path is best here 🤔

Reply:
Ah looks like it is on by default.

Easy enough, if you want that detail you should have the pref selected

Reply:
Oh I see
well since there's an option for it hardcoding kind of a partial overwrite here seems even less favorable?
(I'm not saying the feature couldn't be improved further but it feels weird to "hide" this part here - 
Also kind of takes away the control of the scripter to decide if TgtPrompt$ is needed)
We could change that option to be default in fresh install though?
Kind of makes sense to roll out with the most beginner friendly settings